### DISCUSSION TITLE
Interfaz grafica de Ubunto

### DISCUSSION THREAD

**Discussion by SDagoG19** [nodo-eafit-frontend/frontend-workshop-2024-1#6] [General]
Profe, no se que paso requiero para tener una interfaz grafica en Ubunto como la suya, quisiera que me enseñe porfa

**Comment by DanteCely** [comment]
Resuelto en:
https://github.com/nodo-eafit-frontend/frontend-workshop-2024-f04/discussions/4#discussioncomment-9574086
